src/params.rs

```diff
@@ -278,6 +278,13 @@ impl<T> RangeQuery<T> {
     }
 }
 
+#[derive(Clone, Debug, Serialize)]
+#[serde(untagged)]
+pub enum IdOrCreate<'a, T> {
+    Id(&'a str),
+    Create(&'a T),
+}
+
```

Comment:
Looks like we will need to make this public in the parent module as well. Look like the `params` module is private.

Reply:
Yea, just needs to be added to the `pub use` block on `L81-83` in `src/lib.rs`.

Reply:
Ah yep, good catch